[SDPV-474,490,423] Response Set Curation

kierk opened the pull request:
This addresses the 508 issues, adds response sets to the curation page, and deals with the broken links on the page.

Make sure you include the related JIRA issue in the title e.g. '[SDP-007] Fixed navbar issue'
Make sure you have checked off the following before you issue your Pull Request:

- [x] Added unit tests for new functionality
- [x] Passed all unit tests using `rails test` with 90%+ coverage
- [x] Added cucumber tests for any new functionality
- [x] Passed all cucumber tests using `bundle exec cucumber`
- [x] Passed overcommit hooks, including running all code through Rubocop
- [x] If any database changes were made, run `rake generate_erd` to update the README.md
- [x] If any changes were made to config/routes.rb run `rake jsroutes:generate`
- [x] If any HTML was added or modified check to make sure it was still 508 compliant with WAVE tool / carried over any attributes from similar sections

eedrummer commented:
I think the PR is fine. I have two conflicting thoughts on Section.potential_duplicates method:
* This method will make a lot of calls to elasticsearch. It may make sense to batch them up with https://www.rubydoc.info/gems/elasticsearch-api/Elasticsearch/API/Actions#msearch-instance_method
* It's unfortunate to turn off ABC metrics for the code. potential_duplicates functionality could be pushed into methods in Question and ResponseSet. However, this would make batching up the queries harder.

kierk commented:
@eedrummer As for reducing the number of elasticsearch calls I added a story SDPV-495  to the backlog to reduce load time of the curation wizard on large surveys. This would include switching over to multisearch.
https://publichealthsurveillance.atlassian.net/secure/RapidBoard.jspa?rapidView=6&projectKey=SDPV&view=planning&selectedIssue=SDPV-495

For now I got rid of the ABC Metrics disable and extracted logic to the Question and RS level
